### ISSUE TITLE
Honour manual changes to an overridden attribute (availability, privacy, colour)

### ISSUE THREAD

**Issue by KrisadaFantasy** [phw198/OutlookGoogleCalendarSync#1859]
**Are you already using OGCS?**
2.10.0.0

**Describe the question you have**

I have one-way sync from Google to Outlook calendar. It is read-only department-wide Google shared calendar with a lot of entry not involving me (but I need to be informed about nonetheless) so I set availability to free in sync option. Then I pick the one I need to attend and set it as busy and sometime with reminder in Outlook.

Next sync will rewrote it as free and remove the reminder. Is there any way to exclude specific attribute from being update in sync?

**Comment by phw198** [comment]
For one-way sync G->O, OGCS only syncs items that are modified in Google more recently than Outlook. What is the Google calendar? I'm feeling like you're going to say an ICS subscription by URL....

**Comment by KrisadaFantasy** [comment]
It is indeed subscription by URL... but not the ICS. It is Google calendar at my office shared by public URL (https://calendar.google.com/calendar/embed?src=...) that you can click [+Google Calendar] at the bottom right to add it as read-only to your Google calendar . I added it to my Google calendar that is set to sync with my Outlook calendar. There's always an update on description or attend time so when sync occur and OGCS caught it as entries to be update it also reset my busy and reminder in Outlook.

**Comment by phw198** [comment]
In your opening post you said "Next sync will rewrote it" but now you're saying whenever there is a legitimate change of date or description, then OGCS updates Outlook?

The latter scenario makes sense (at first it sounded like everything was being updated in Outlook for every sync). When you configure OGCS to set the items as Free, is that for all items synced or only items created? You'll want to choose the second option to stop that being changed on modification. For reminders, just don't sync them?

(It's always easier to troubleshoot these things if your ogcalsync.log file can be provided.)

**Comment by KrisadaFantasy** [comment]
It's mistake on my part and I am sorry for the misunderstanding. At first all the entries I had set to busy got rewrote as free because by chance all such entries had updates. After reading your first comment and reviewed new sync it was indeed only rewrote busy status I set in Outlook if new change was made on Google side and the entry get update.

![image](https://github.com/phw198/OutlookGoogleCalendarSync/assets/85682504/acb4167f-e2d1-4b42-99a2-9e3a05b041da)

The option is locked as [all items.] I only set location and description to sync and not the reminder. I don't think there're reminders from this calendar. I guess that why it rewrite my Outlook calendar to source's "no reminder." I understand that it should not sync reminder at all if I choose not to sync it as you suggest but it remove my reminder nonetheless.

I need to sync all the modification from source not just new entries so I get the latest update on work schedule, but I want to be able to personalise it on Outlook with status and reminder without risking it being wipe out the next sync.

I have included the log I found it change both availability and reminder on line 229 as
`2024-05-16 07:10:43,811 DEBUG  23 CalendarSync.Console [294] -  AttributeChangeFree/BusyolBusy → olFreeReminder15 → removed` Thre's a few more entries that change busy to free as well.

[OGcalsync.log.2024-05-16.txt](https://github.com/phw198/OutlookGoogleCalendarSync/files/15346489/OGcalsync.log.2024-05-16.txt)

I'm sorry for the inconvenience and thank you for your time on this issue.

**Comment by phw198** [comment]
> The option is locked as [all items.]

Ah yes, you are quite correct. Though thinking about it, this doesn't seem necessary and I can make this work how you need.

:package: Please have a go with [hotfix v2.10.4.9.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/15367033/v2.10.4.9.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [CRCs](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Known-Issues#cyclic-redundancy-checks---aka-crcs): Zip = `EA3F7DD9` Exe = `6E9AA7E7`

Let me know how it goes...

**Comment by KrisadaFantasy** [comment]
I had applied hotfix on top of 10.2.4 and changed setting from [all items] to [items created]. I go to Outlook, set entries to busy and reminder to 15 minutes, then wait for auto sync and my workplace to wake up on Monday and start changing things.

I can confirm that availability is not longer overwrote from Google! But the reminder is still removed from my Outlook.

[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/15373134/OGcalsync.log)

**Comment by phw198** [comment]
The way reminders sync has been a little unclear for a while and is a separate issue - see #1458 which I have just reopened to track this again.

I'll need to give this some though on how to differentiate "do not sync reminders because I don't want OGCS to change them" vs "do not sync reminders because I don't want to be receive any reminder notifications" - the latter is how OGCS works and why it removes existing reminders.

**Comment by KrisadaFantasy** [comment]
> do not sync reminders because I don't want to be receive any reminder notifications

I admit that it confuse me when I set my up profile. I thought set OGCS to not sync means it will skip looking into reminder altogether. Right now "do not sync" is equal to "remove all reminder" because "I don't want to be receive any reminder notifications."

In my case, the source calendar has no reminder. I took that as "no reminder" status. Now I set reminder in my target Outlook calendar, "do not sync" is syncing "no reminder" status and overwrite my reminders. In my opinion, "do not sync reminder" should work as your first case "because I don't want OGCS to change them". And if "I don't want to be receive any reminder notifications" then there should be an option to set everything to sync as "no reminder."

But I guess the latter will have the problem like this overridden availability issue...
